<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventQueryDialog</class>
 <widget class="QDialog" name="EventQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="timeBetweenRadioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Between start and end times</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="timeWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <widget class="QDateTimeEdit" name="endtimeDateTimeEdit">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>12</y>
           <width>144</width>
           <height>24</height>
          </rect>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
        <widget class="QDateTimeEdit" name="starttimeDateTimeEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>12</y>
           <width>160</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
        <widget class="QLabel" name="starttimeLabel">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>12</y>
           <width>40</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>start</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="endtimeLabele">
         <property name="geometry">
          <rect>
           <x>255</x>
           <y>12</y>
           <width>40</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>end</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="timeDuringStationsRadioButton">
        <property name="text">
         <string>During operation of selected stations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="magnitudeGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="title">
      <string>Magnitude</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="magnitudeWidget" native="true">
        <widget class="QLineEdit" name="minmagLineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="maxmagLineEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
        <widget class="QComboBox" name="magtypeComboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>190</width>
           <height>26</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>All Types</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ML</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MW</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>12</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="depthGroupBox">
     <property name="title">
      <string>Depth</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="depthWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <widget class="QLineEdit" name="mindepthLineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="maxdepthLineEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>6000</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>12</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="locationGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>375</height>
      </size>
     </property>
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="locationRangeRadioButton">
        <property name="text">
         <string>Within lat/lon box</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="locationRangeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <widget class="QLineEdit" name="locationRangeEastLineEdit">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="locationRangeWestLineEdit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="locationRangeNorthLineEdit">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="locationRangeSouthLineEdit">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="locationDistanceFromPointRadioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Distance from point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="distanceFromPointWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <widget class="QLineEdit" name="distanceFromPointEastLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>40</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>E</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>225</x>
           <y>40</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>N</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="distanceFromPointMinRadiusLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>10</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="distanceFromPointMaxRadiusLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="distanceFromPointNorthLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>165</x>
           <y>10</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>degrees</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>40</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>from</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <zorder>label_10</zorder>
        <zorder>distanceFromPointMaxRadiusLineEdit</zorder>
        <zorder>distanceFromPointMinRadiusLineEdit</zorder>
        <zorder>label_11</zorder>
        <zorder>distanceFromPointEastLineEdit</zorder>
        <zorder>label_8</zorder>
        <zorder>label_9</zorder>
        <zorder>distanceFromPointNorthLineEdit</zorder>
        <zorder>label_6</zorder>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="locationDistanceFromStationsRadioButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Distance from selected stations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="distanceFromStationsWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <widget class="QLineEdit" name="distanceFromStationsMinradiusLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>12</y>
           <width>10</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="distanceFromStationsMaxradiusLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>12</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>165</x>
           <y>12</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>degrees</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
